apt-hook/Makefile

```diff
@@ -6,7 +6,7 @@ ubuntu-advantage.pot: hook.cc
 	xgettext hook.cc -o ubuntu-advantage.pot
 
 hook: hook.cc
-	$(CXX) -Wall -Wextra -pedantic $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -g -o hook hook.cc -lapt-pkg $(LDLIBS)
+	$(CXX) -Wall -Wextra -pedantic -std=c++11 $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -g -o hook hook.cc -lapt-pkg $(LDLIBS)
```

Comment:
hrm does this mean a package build depedency  on libc++abi1-11? If so, we need to check rmadison
```bash
$  rmadison libc++abi1-11
 libc++abi1-11 | 1:11.0.0-2~ubuntu20.04.1 | focal-updates/universe | amd64, arm64, armhf, i386, ppc64el, riscv64, s390x
 libc++abi1-11 | 1:11.0.0-2               | groovy/universe        | amd64, arm64, armhf, i386, ppc64el, riscv64, s390x
 libc++abi1-11 | 1:11.0.1-2ubuntu2        | hirsute/universe       | amd64, arm64, armhf, i386, ppc64el, riscv64, s390x
```

I see no C++ 11 in Xenial or trusty, so that would mean we can no longer build this hook on either right?
We can test w/ `sed -i s/UNRELEASED/xenial' debian/changelog; git commit -am 'wip'; build-package; sbuild-it ../out*27.0.dsc;

Reply:
This did build successfully for me. But I also understand if we'd rather play it safe and remove the dependency. I can replace `std::array` with `std::vector` and replace `std::to_string` with some `stringstream` stuff - its not hard, just not as clean-looking.

Reply:
@orndorffgrant thanks I confirmed too package builds on trusty/xenial succeed and the binary gerated works. So as long as we have integration tests covering the template rendering output we should be good on this as is.